kal-aalrajhi/overlook-hotel
# Feature/book room

## Description 
Users are now able to book a room and have it persist, even if user restarts the browser. This is all able to be done because we've integrated FETCH and it's respective GET/POST HTTP methods. Anytime a user adds a booking it's posted up to the booking API and linked to the user. Some classes needed to be modified to fit this model, thus we've had to refactor class tests and their respective classes.

#### How to test
To run the server used, locally, please clone and [follow the instructions from this repo](#https://github.com/turingschool-examples/overlook-api).

run `npm start`

navigate to the *book* view and begin booking your stay! Switch back to your dashboard and see that new bookings have been added.

#### Type of Change:

- [x] Refactor (code is working and without bugs, formatting has been streamlined)


#### How Has This Been Tested?

What steps have you taken to ensure the code is effective?

#### Checklist:
